pkgs/by-name/gp/gpth/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildDartApplication,
+  fetchFromGitHub,
+}:
+
+buildDartApplication rec {
+  pname = "gpth";
+  version = "v3.4.3";
```

[version must start with a digit](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning)

```suggestion
  version = "3.4.3";
```